### PR TITLE
Add some basic navigation

### PR DESCRIPTION
**Pull request by lbkchen**
Uses https://github.com/expo/vector-icons. Refer to this directory to search for icons: https://expo.github.io/vector-icons/. 

img_2312

### CODE REVIEW

**Comment by nikitr** [comment]
LGTM!!

File: src/helpers/requester.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+class Request {
```
Comment (amillman) [inline src/helpers/requester.js:1]: dont use / copy this from normal react projects. there is a built in mozilla library that cleanly handles requests called `fetch`, and you can make a simpler wrapper around that. we use it in https://github.com/calblueprint/replate-marketplace/tree/andrew/login/requesters

Comment (lbkchen) [reply src/helpers/requester.js:1]: Ye thanks! I was actually gonna do that as well. @nikitr I’ll put it in another pr before GM

Comment (nikitr) [reply src/helpers/requester.js:1]: Perfect, thanks!

**Pull request merged**